packages/cloud_firestore/cloud_firestore_web/lib/src/interop/utils/utils.dart

```diff
@@ -23,8 +24,7 @@ dynamic dartify(Object? jsObject) {
       return object;
     }
     if (util.instanceof(object, TimestampJsConstructor)) {
-      return DateTime.fromMillisecondsSinceEpoch(
-          (object as TimestampJsImpl).toMillis());
+      return Timestamp((object as TimestampJsImpl).seconds, object.nanoseconds);
```

Comment:
Does that mean what used to be decoded as a DateTime is now decoded as a Timestamp? If so, that could be a breaking change

Reply:
We were previously decoding it as follow TimestampJS => DateTime => Timestamp
Now we only do TimestampJS => Timestamp

So no breaking change :)

Reply:
I think it would also show up in the e2e tests if it was a [problem](https://github.com/firebase/flutterfire/blob/master/packages/cloud_firestore/cloud_firestore/example/test_driver/timestamp_e2e.dart#L36)